tests/unit/test_visionEnhancementProviders/test_magnificationAPI.py

```diff
@@ -0,0 +1,49 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2024 NV Access Limited.
+
+"""Unit tests for the magnification Windows API."""
+
+import unittest
+
+from visionEnhancementProviders.screenCurtain import Magnification, TRANSFORM_BLACK
+
+
+class Test_ScreenCurtain(unittest.TestCase):
+	def setUp(self):
+		self.assertTrue(Magnification.MagInitialize())
+
+	def tearDown(self):
+		self.assertTrue(Magnification.MagUninitialize())
```

Comment:
Can the setup and teardown methods be put in a common base class?

Reply:
I'd prefer not to as I expect the setup and tear down to potentially diverge as more tests are added

Reply:
actually, that will only really affect the magnifier test classes, not these